internal/controller/kyma/controller.go

```diff
@@ -115,41 +114,71 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}
 
 	if r.SyncKymaEnabled(kyma) {
-		err := r.SkrContextFactory.Init(ctx, kyma.GetNamespacedName())
-		if !kyma.DeletionTimestamp.IsZero() && errors.Is(err, remote.ErrAccessSecretNotFound) {
-			logger.Info("access secret not found for kyma, assuming already deleted cluster")
-			r.Metrics.CleanupMetrics(kyma.Name)
-			r.removeAllFinalizers(kyma)
-
-			if err := r.updateKyma(ctx, kyma); err != nil {
-				r.Metrics.RecordRequeueReason(metrics.KymaUnderDeletionAndAccessSecretNotFound, queue.UnexpectedRequeue)
-				return ctrl.Result{}, err
-			}
-			r.Metrics.RecordRequeueReason(metrics.KymaUnderDeletionAndAccessSecretNotFound, queue.IntendedRequeue)
-			return ctrl.Result{Requeue: true}, nil
+		result, err := r.handleRemoteInitialization(ctx, kyma)
+		if err != nil {
```

Comment:
you didn't catch this case `return ctrl.Result{Requeue: true}, nil`

Reply:
I mean [here](https://github.com/kyma-project/lifecycle-manager/pull/1426/files/834af8e799d6a3cf317c9898980788141aa0a8a1..b1dad117f6ef192357ecc852d7b3a42026af0671#diff-4d414901850a616d4e06a120615896904639aef683f0a4d8860abf65c76179d4R139)